backend/tests/alert.test.js

```diff
@@ -16,8 +24,15 @@ jest.mock('../logError', () => ({
 config.app.notmEndpointUrl = '//newNomisEndPointUrl/'
 
 describe('alert management', () => {
-  const res = { render: jest.fn(), redirect: jest.fn(), locals: {} }
-  const mockReq = { flash: jest.fn().mockReturnValue([]), originalUrl: '/close-alert/', get: jest.fn(), body: {} }
+  let mockReq
+  let res
```

Comment:
The request and response were not resetting after each test, resulting in the test rendering the wrong template even though the right controller was being called.